Zapiski/ugtzapiski.tex

```diff
@@ -571,7 +571,11 @@ \subsection{Kvocientne preslikave}
   Na $B^2$ imamo ekvivalenčno relacijo, v kateri je edini netrivialni ekvivalenčni razred $S^1$:
     to lahko označimo kot $\quot{B^2}{S^1}$ in že intuitivno se zdi, da je ta prostor homeomorfen $S^2$.
     Možen predpis za kvocientno preslikavo $f: B^2 \to S^2$ bi bil
-    $f(\vb{x}) = ((2 \sqrt{1 - \|\vb{x}\|^2}) \vb{x}, 2\|\vb{x}\|^2 - 1)$.
+    $$f(\vb{x}) = \begin{cases}
+        ((2 \sqrt{1 - \|\vb{x}\|^2}) \frac{\vb{x}}{\|\vb{x}\|}, 2\|\vb{x}\|^2 - 1);\, \vb{x} \neq \vb{0} \\
```

Comment:
Tole je narobe. Ostali popravki so kul, ampak predpis ki je bil na začetku je v redu. Če vzameš kvadrat norme prve komponente in kvadrat druge komponente, dobiš, da se seštejeta v 1. Tvoja funkcija ne slika na sfero.